pages/home/addTask.js

```diff
@@ -39,12 +42,16 @@ export default function AddTask() {
   async function handleSubmit(event) {
     event.preventDefault();
     await submitTask();
-    setPopup(!popup);
+    triggerPopup();
   }
 
-  const resetPopup = () => {
-    setPopup(false);
-  };
+  function triggerPopup() {
+    if (popup == true) {
+      setPopup(false);
+    } else {
+      setPopup(true);
+    }
+  }
```

Comment:
I think you don't need this function. Take a look at my comment above where you call this function.